test/functional/desired.js

```diff
@@ -78,6 +78,7 @@ const SAFARI_CAPS = amendCapabilities(GENERIC_CAPS, {
 
 const TESTAPP_CAPS = amendCapabilities(GENERIC_CAPS, {
   'appium:app': APPS.iosTestApp,
+  'appium:noReset': false,
```

Comment:
I wouldn't do it as other tests could be affected

Reply:
OK, I wanted to clean the app each time, then I added terminateApp to eachBefore instaed. 
https://github.com/appium/appium-xcuitest-driver/pull/1461/commits/9a1fa736a4234d989c9846f522d6c9212bc3028d